images/win/scripts/Installers/Validate-Go.ps1

```diff
@@ -8,18 +8,25 @@ function Get-GoVersion
 {
     Param
     (
-        [String]$goRootPath
+        [String]$goVersion
     )
+    $goDirectory = Get-ChildItem -Path $env:SystemDrive -Force -Filter "Go$goVersion*" | Select-Object -First 1
```

Comment:
Why we need -Force flag for Get-ChildItem?

Reply:
Done.